data/human/transport missions.txt

```diff
@@ -3209,3 +3209,76 @@ mission "Earth Retirement"
 			label reject
 			`	Charles looks surprised for a moment, and then, for the first time you've seen, he smiles. "Thank you, Captain <last>. I wish there were more people like you in the Paradise Worlds."`
 			`	You wish the couple the best of luck on Earth, and they slowly make their way to the train station. Presumably, their new apartment is far from here.`
+
+
+
+mission "Spacediving professionals"
+	name "Spacediving at <destination>"
+	description "Drop skydivers from the edge of space. They are carrying cooled spacesuits, durable parachutes and oxygen bottles to ensure the survival of their party. Payment is <payment>."
+	passengers 5 10
+	repeat
+	job
+	source
+		government "Republic" "Free Worlds" "Syndicate"
+	destination
+		distance 7 9
+		government "Republic" "Free Worlds" "Syndicate"
+	to offer
+		and
+			credits >= 1000000
+			random > 96
+	on visit
+		dialog phrase "generic passenger on visit"
+	on complete
+		payment 8000
+		dialog "The skydivers jump from your ship and make a spectacular dive until they open their parachutes and land safely near the spaceport. You received payment of <payment> just before they jumped out."
+
+
+
+mission "Spacediving amateurs"
+	name "Spacediving at <destination>"
+	description "Drop skydivers from the edge of space. They are carrying cool wingsuits, fashionable parachutes and beer bottles to ensure the trip will be a great party. Payment is 8000 credits."
+	passengers 5 10
+	repeat 3
```

Comment:
Why the limit?

Reply:
I moved the limit only to the job with the fine. It doesn't seem right to bother the player with multiple fines (such joke is only funny once).